src/librustc_typeck/check/writeback.rs

```diff
@@ -658,6 +669,7 @@ impl<'cx, 'tcx> TypeFolder<'tcx> for Resolver<'cx, 'tcx> {
                 debug!("Resolver::fold_const: input const `{:?}` not fully resolvable", ct);
                 // FIXME: we'd like to use `self.report_error`, but it doesn't yet
                 // accept a &'tcx ty::Const.
+                self.replaced_with_error = true;
```

Comment:
I think the comment is now handled by this change?

Reply:
No, `report_error` is for an user-facing error message.

Reply:
I see, I got confused when I saw `self.tables.tainted_by_errors |= resolver.replaced_with_error;` and I thought this referred to that.